### PR TITLE
Update "Downloading a binary from GitHub Releases" section

### PR DESCRIPTION
**Pull request by boraoren**
add directory parameter for unzip operation.

```release-note
NONE
```

### CODE REVIEW

**Comment by kron4eg** [comment]
@boraoren thank for the PR!

/lgtm
/approve

**Comment by kron4eg** [comment]
@boraoren apparently your PR targets wrong branch (should be master).

File: README.md [hunk 1]
```diff
@@ -56,21 +56,24 @@ releases page, find the binary for your operating system
 and architecture and download it or grab the URL and use
 `wget` or `curl` to download the binary.
 
+**version :** version of kubeone  
+**operation system :** operation system of kubeone (`linux`, `darwin` for mac)
```
Comment (xmudrii) [inline README.md]:
```suggestion
**Operating system:** `linux` or `darwin` for macOS
```

File: README.md [hunk 2]
```diff
@@ -56,21 +56,24 @@ releases page, find the binary for your operating system
 and architecture and download it or grab the URL and use
 `wget` or `curl` to download the binary.
 
+**version :** version of kubeone
```
Comment (xmudrii) [inline README.md]:
```suggestion
**Version:** version of KubeOne  
```

File: README.md [hunk 3]
```diff
@@ -56,21 +56,24 @@ releases page, find the binary for your operating system
 and architecture and download it or grab the URL and use
 `wget` or `curl` to download the binary.
 
+**version :** version of kubeone  
+**operation system :** operation system of kubeone (`linux`, `darwin` for mac)
+
 ```bash
-curl -LO https://github.com/kubermatic/kubeone/releases/download/v<version>/kubeone_<version>_linux_amd64.zip
+curl -LO https://github.com/kubermatic/kubeone/releases/download/v<version>/kubeone_<version>_<operation system>_amd64.zip
```
Comment (xmudrii) [inline README.md]:
```suggestion
curl -LO https://github.com/kubermatic/kubeone/releases/download/v<version>/kubeone_<version>_<operating_system>_amd64.zip
```

File: README.md [hunk 4]
```diff
@@ -56,21 +56,24 @@ releases page, find the binary for your operating system
...
 ```
 
-Extract the binary. On Linux and macOS, you can use `unzip`.
+Extract the binary to kubone directory. On Linux and macOS, you can use `unzip`.
```
Comment (xmudrii) [inline README.md]:
```suggestion
Extract the binary to the KubeOne directory. On Linux and macOS, you can use `unzip`.
```

File: README.md [hunk 5]
```diff
@@ -56,21 +56,24 @@ releases page, find the binary for your operating system
...
 
 ```bash
-unzip kubeone_0.8.0_linux_amd64.zip
+unzip kubeone_<version>_<operation system>_amd64.zip -d kubeone_<version>_<operating system>_amd64
```
Comment (xmudrii) [inline README.md]:
```suggestion
unzip kubeone_<version>_<operating_system>_amd64.zip -d kubeone_<version>_<operating_system>_amd64
```

File: README.md [hunk 6]
```diff
@@ -56,21 +56,24 @@ releases page, find the binary for your operating system
...
 ```
 
 Move the `kubeone` binary to your path, so you can easily
 invoke it from your terminal.
 
 ```bash
-sudo mv kubeone /usr/local/bin
+sudo mv kubeone_<version>_<operating system>_amd64/kubeone /usr/local/bin
```
Comment (xmudrii) [inline README.md]:
```suggestion
sudo mv kubeone_<version>_<operating_system>_amd64/kubeone /usr/local/bin
```

**Pull request closed without merging**